sphinx/texinputs/sphinx.sty

```diff
@@ -517,17 +533,24 @@
     % define chaptermark with \@chappos when \@chappos is available for Japanese
     \ltx@ifundefined{@chappos}{}
       {\def\chaptermark##1{\markboth{\@chapapp\space\thechapter\space\@chappos\space ##1}{}}}
-  }
+    }
   % Update the plain style so we get the page number & footer line,
   % but not a chapter or section title.  This is to keep the first
   % page of a chapter and the blank page between chapters `clean.'
```

Comment:
I do not understand the "blank page between chapters" bit from this legacy comment. In report class (used by 'manual'), the `\chapter` command does indeed issue `\thispagestyle{plain}` after having done `\cleardoublepage`,  but there is no modification of the header/footer style for the last page of previous chapter, if the latter has no contents. Only the first page of new chapter is re-styled via the `\thispagestyle{plain}` arising from `\chapter`.

Thus I should probably remove this old comment which I believe to be misleading.